### PR TITLE
Fix Table Overflow

### PR DESCRIPTION
**Pull request by NPDebs**
This PR fixes issue #420.

The width of the table had caused other elements to have too much space on the right.
My solution was to write media queries using breakpoints between 768px to 320px. A reduction in `font-size` (and slight increase in `line-height`, to ensure readability), seemed to do the trick.

Additionally, this approach fixed alignment issues across other pages containing tables.

ols-pr-resp

<br />

ols-pr-resp2

### CODE REVIEW

**Comment by NPDebs** [comment]
@bebatut & @yochannah
Do kindly review when you get the chance. Thanks!

_Please, note that I made this PR on a different branch (of my fork) to prevent any merge conflicts._

File: css/custom.scss [hunk 1]
```diff
@@ -59,7 +59,7 @@ $link-active: $link-hover;
   background-position: center center;
   background-repeat: no-repeat;
   background-size: cover;
-  height: 450px;
+  height: auto;
```
Comment (bebatut) [inline css/custom.scss:62]: This change belongs to your other PR I think

**Review by bebatut — COMMENTED** [review]
Nice @NPDebs

Sadly, your change make the table hard to read on small screen: 

image

**Comment by NPDebs** [comment]
> Nice @NPDebs
> 
> Sadly, your change make the table hard to read on small screen:
> 
> image

Ah, yes. That's true. 🤦🏽‍♀️
I tried to salvage that by increasing the line height. Apparently, that didn't help much.

What do you recommend, please?

File: css/custom.scss (hunk 1 above)
Comment (bebatut) [reply css/custom.scss:62]: yes it would be great

Comment (NPDebs) [reply css/custom.scss:62]: Done. Thank you. 🙌🏼

**Comment by NPDebs** [comment]
Hi, @bebatut Eureka!
Instead of shrinking the font-size and unreadable extent, I added a horizontal scrollbar to the table. The padding at the bottom is to separate the content from the scrollbar.

Do let me know your thoughts on this approach. Thank you!

ols-pr-scroll

**Comment by bebatut** [comment]
Good idea the scrollbar.

But it somehow broke the header:

image
image

**Comment by NPDebs** [comment]
Hi @bebatut. Thanks for the review.

Everything worked fine when I ran the site locally.
So, on commenting out the code, I noticed the problem had nothing to do with the scrollbar. 

Earlier, I had accidentally edited line 65 (instead of 64), changing the header width from 100% to `width: 450px`.
In other to avoid a merge conflict with my other PR, I was going to change the height from `auto` to `450px`...

In summary: Crisis averted. Everything looks good on this end. 👍
Please review when you get the chance. Thanks again! 🙌 

Screenshot (295)

**Review by bebatut — APPROVED** [review]
It looks good now. Thanks @NPDebs

**Comment by NPDebs** [comment]
> It looks good now. Thanks @NPDebs 

Thanks for the detailed review and the merge. 🙌🏼🤸🏽‍♀️🤍 @bebatut
